space.go

```diff
@@ -138,6 +147,28 @@ func importSpaceV2(source map[string]interface{}) (*space, error) {
 	}, nil
 }
 
+func importSpaceV3(source map[string]interface{}) (*space, error) {
+	fields, defaults := spaceV2Fields()
```

Comment:
Don't do this here - add a spaceV3Fields() method.
I see that V2 used this pattern - whomever did it originally did it wrong :-)